dosish.h

```diff
@@ -49,7 +49,7 @@
 
 #ifndef PERL_SYS_TERM_BODY
 #  define PERL_SYS_TERM_BODY()                         \
-    HINTS_REFCNT_TERM; KEYWORD_PLUGIN_MUTEX_TERM;      \
+    SHUTDOWN_TERM; HINTS_REFCNT_TERM; KEYWORD_PLUGIN_MUTEX_TERM;      \
```

Comment:
Most minor of nitpicks, but can we reflow these words and reälign the resulting `\` escapes, to keep it neat?

Reply:
We could, but to be honest I was about to open a topic about restructuring `PERL_SYS_TERM_BODY` entirely. There are half a dozen definitions of it, but they're 90% identical. That seems suboptimal.